modules/performing-an-image-signature-verification.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * operating/verify-image-signatures.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="performing-an-image-signature-verification_{context}"]
+= Performing an image signature verification
+
+Before you perform an image signature verification, you must first create a signature integration in {rh-rhacs-first}.
+
+When you create a signature integration, you can use the following verification methods:
+
+* Cosign public keys
+* Cosign certificates
+* Transparency log
+
+.Prerequisites
+
+. You have a Cosign public key which is encoded in Privacy Enhanced Mail (PEM) format.
++ 
+For more information about Cosign public keys, see link:https://docs.sigstore.dev/cosign/signing/overview/[Overview] (Sigstore documentation).
+
+. You have the certificate identity and issuer.
+. Optional: You have a certificate and chain which is encoded in PEM format. 
```

Comment:
```suggestion

. Optional: You have a certificate and chain which are encoded in PEM format. 
```
There should be a line space between steps. Also, are both certificate and chain encoded? If so, the verb is "are." If not, revise to "You have a certificate. You also have a chain which is encoded in PEM format." (or something like that)